catboost/python-package/catboost/core.py

```diff
@@ -5826,3 +5826,71 @@ def to_classifier(model):
     for attr in model.__dict__:
         setattr(classifier, attr, getattr(model, attr))
     return classifier
+
+
+from catboost.utils import read_cd
+
+
+class ColumnDescription:
```

Comment:
Let's add internal class `Column` and a possibility to iterate over columns in `ColumnDescription`, like:
```
for column in column_description:
    print(column.index, column.type, column.name)
```